Makefile

```diff
@@ -629,3 +634,10 @@ clean-temporary: ## Remove all temporary files and folders
 	rm -rf test/e2e/capi-kubeadm-control-plane-controller-manager
 	rm -rf test/e2e/logs
 	rm -rf test/e2e/resources
+
+##@ hack/tools:
+
+go-apidiff: $(GO_APIDIFF) ## Build a local copy of apidiff
+
+$(GO_APIDIFF): $(TOOLS_DIR)/go.mod # Build go-apidiff from tools folder.
```

Comment:
How about moving this to hack/tools?

Reply:
Just checked, we already have this target there:

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/24e3ea2e372e5c6deff3c32bb6e72e4207f0f24a/hack/tools/Makefile#L67-L68

Reply:
Removed these lines from the change.